main/models.py

```diff
@@ -16,6 +16,10 @@ class DataSourceMember(models.Model):
     """
     user = models.OneToOneField(OpenHumansMember, on_delete=models.CASCADE)
     # Your other fields should go below here
+    github_id = models.CharField(max_length=255, unique=True, null=True)
+    access_token = models.CharField(max_length=512, null=True)
+    scope = models.CharField(max_length=512, null=True)
+    token_type = models.CharField(max_length=512, null=True)
```

Comment:
I've added the necessary models here. I left off `refresh_token` and `token_expires` because Github OAuth tokens [do not expire](https://stackoverflow.com/questions/26902600/whats-the-lifetime-of-github-oauth-api-access-token). The OAuth app administrator can revoke or invalidate tokens, but they do not explicitly expire on their own.